lib/bfd.h

```diff
@@ -473,6 +473,9 @@ extern bool bfd_protocol_integration_debug(void);
  */
 extern bool bfd_protocol_integration_shutting_down(void);
 
+extern void bfd_nht_zclient_connected(struct zclient *zclient);
```

Comment:
Please write documentation for this now exported functions.

`bfd_nht_zclient_connected` should be called on zebra client socket reconnection and `bfd_nht_update` should be called for zebra next hop tracking updates.

Reply:
bfd_nht_zclient_connected is removed in the second commit.